RELEASE_NOTES.md

```diff
@@ -2,85 +2,58 @@
 
 ## Summary
 
-This version ships an experimental version of the **Power Manager**, adds preliminary support for n:m relations between inverters and batteries and includes user documentation.
+The `microgrid` package now exposes grid connections uniformly and introduces formula operators for `consumption` and `production`, replacing the `logical_meter.*_{production,consumption}()` formulas. The `actor` package restarts crashed actors with a delay, and the `ConnectionManager` exposes the `microgrid_id` and `location` details.
+
+There are also a few bug fixes, documentation improvements and other minor breaking changes.
 
 ## Upgrading
 
-- `microgrid.battery_pool()` method now accepts a priority value.
+- `actor` package
 
-- `microgrid.grid()`
+  * Actors are now restarted after a small delay when they crash to avoid a busy loop and spamming the logs if the actor keeps failing to start.
 
-  * Similar to `microgrid.battery_pool()`, the Grid is now similarily accessed.
+- `microgrid` package
 
-- `BatteryPool`'s control methods
+  * `grid`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
```

Comment:
If you'll update this patch, I'd suggest to use `grid()` to be consistent with `logical_metter()`

```suggestion
  * `grid()`: The grid connection is now exposed as `microgrid.grid()`. This is more consistent with other objects exposed in the `microgrid` module, such as `microgrid.battery_pool()` and `microgrid.logical_meter()`.
```

Reply:
I intentionally used the old name here, because it was renamed.